airbyte-protocol/models/src/main/resources/airbyte_protocol/airbyte_protocol.yaml

```diff
@@ -171,6 +176,14 @@ definitions:
         type: array
         items:
           type: string
+      alias_name:
```

Comment:
why did you decide to drop the destination and go for alias & target? I have a preference for the destination prefix

Reply:
No particular reason, I reflected on it and thought it would make it clearer what it is used for afterward... but we can always revert to `destination_name` / `destination_namespace` instead If you think it's better